app/controllers/guides_controller.rb

```diff
@@ -147,8 +147,8 @@ def guide_params(with = {})
         :related_discussion_href,
         :related_discussion_title,
         :update_type,
+        :change_reason_for_change,
```

Comment:
Ha :)

Maybe it'd be easier if these columns didn't match the schema fields. `change_summary` and `change_description` and we map them in the presenter?